### PR TITLE
tetragon: logging improvements for non k8s deployment

### PR DESCRIPTION
**Pull request by tixxdz**
Signed-off-by: Djalal Harouni <tixxdz@gmail.com>

### CODE REVIEW

File: cmd/tetragon/main.go [hunk 1]
```diff
@@ -321,9 +321,26 @@ func tetragonExecute() error {
 		}
 	}
 
+	// Explicitly log status every 1 hour
+	go logStatus(ctx, obs, 1*time.Hour)
```
Comment (kkourt) [inline cmd/tetragon/main.go:325]: Do we need to make this configurable? Is so, let's split the two patches and do two different PRs.

Comment (willfindlay) [reply cmd/tetragon/main.go:325]: +1 on making this configurable

Comment (tixxdz) [reply cmd/tetragon/main.go]: Ok will split it in separate patch and think about making it configurable, I'm not a fan of adding too much options better try to do the right thing by default

**Review by kkourt — APPROVED** [review]

**Review by michi-covalent — APPROVED** [review]
